contrib/ansible/openshift/register.sh

```diff
@@ -33,7 +33,9 @@ while read -u 9 line;do
   remote_name=$(echo $line | awk -F' ' '{print $1}')
   group_name=$(echo $line | awk -F' ' '{print $2}')
   index_value=$(echo $line | awk -F' ' '{print $3}')
+  index_value=$(( index_value + 1 )) 
   label_name=$(echo $line | awk -F' ' '{print $4}')
+  label_name="$label_name"_"$index_value"
```

Comment:
It would be great to have a comment here to just note that the final label includes the index number to make it unique, or something to that effect.